test/test_custom.py

```diff
@@ -85,18 +94,39 @@ def test_round_trip(self):
         obj.foo_bool.append(True)
         obj.foo_bool.append(False)
         self.assertEqual([True, False], obj.foo_bool)
+        obj_unregistered_name = 'bool_test_unregistered'
+        obj_u = CustomUnregisteredTopClass(obj_unregistered_name)
+        obj_u.foo_bool.append(True)
+        self.assertEqual([True], obj_u.foo_bool)
         doc = sbol3.Document()
         doc.add(obj)
+        doc.add(obj_u)
         doc2 = sbol3.Document()
         # Round trip the document
         with tempfile.TemporaryDirectory() as tmpdirname:
             test_file = os.path.join(tmpdirname, 'custom.nt')
             doc.write(test_file, sbol3.NTRIPLES)
             doc2.read(test_file, sbol3.NTRIPLES)
         obj2 = doc2.find(obj_name)
+        obj_u2 = doc2.find(obj_unregistered_name)
         # The lists are necessarily unordered because of RDF
-        # Compare specially
-        self.assertCountEqual([True, False], obj2.foo_bool)
+        self.assertEqual([False, True], sorted(obj2.foo_bool))
```

Comment:
Why not stick with `assertCountEqual`? I think [assertCountEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual) does exactly what you want here.

Reply:
You are correct --- I was misled by my unfamiliarity with the function and thinking its name meant that it was checking it the lengths were equal. I have switched to assertCountEqual both here and below.